.github/workflows/pypi-cd.yml

```diff
@@ -43,7 +43,7 @@ jobs:
                 # Get the matrix from the matrix.json file, without the object that has the IMAGE key
                 export "PLATFORM_MATRIX=$(jq 'map(select(.PACKAGE_MANAGERS | contains(["pypi"])))' < .github/json_matrices/build-matrix.json | jq -c .)"
                 echo "PLATFORM_MATRIX=${PLATFORM_MATRIX}" >> $GITHUB_OUTPUT
-                
+
```

Comment:
Probably, you can revert all whitespace changes in other workflows and do them in another PR